schema/common/defaults/configuration/image-registry.yml

```diff
@@ -47,6 +47,16 @@ specification:
           file_name: kube-proxy-v1.19.15.tar
         - name: "k8s.gcr.io/kube-scheduler:v1.19.15"
           file_name: kube-scheduler-v1.19.15.tar
+        # K8s v1.20.12 - Epiphany 1.3
```

Comment:
Now we have both 1.19 and 1.20 under `current`. I would move 1.19 under `legacy` to not being loaded in `apply` mode (to save time). Maybe the `legacy` key is not informative enough and could be renamed to `upgrade_mode`?

Reply:
For me `legacy` is clear, it was a copy-paste mistake. Thanks.